tools-local/update-submodules.targets

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask AssemblyFile="$(TasksBinDir)Microsoft.DotNet.SourceBuild.Tasks.dll" TaskName="ReadGitConfigFile" />
+
+  <!-- Create a git-like alias for UpdateToRemoteDependencies. -->
+  <Target Name="PullSubmodules" DependsOnTargets="UpdateToRemoteDependencies" />
+
+  <Target Name="CreateSourceBuildSubmoduleUpdateSteps"
+          BeforeTargets="CreateDefaultDependencyInfos">
+    <ReadGitConfigFile File="$(ProjectDir).gitmodules">
+      <Output TaskParameter="SubmoduleConfiguration" ItemName="SubmoduleConfiguration" />
+    </ReadGitConfigFile>
+
+    <PropertyGroup>
+      <TargetSubmodule Condition="'$(TargetSubmodule)' != '' AND !$(TargetSubmodule.StartsWith('src'))">src/$(TargetSubmodule)</TargetSubmodule>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- By default, upgrade each submodule with a branch configured in '.gitmodules'. -->
+      <SubmoduleConfiguration>
+        <SourceBuildAutoUpdate Condition="'%(SubmoduleConfiguration.SourceBuildAutoUpdate)' == ''">true</SourceBuildAutoUpdate>
+      </SubmoduleConfiguration>
+
+      <_UpgradableSubmodule Include="@(SubmoduleConfiguration)"
+                            Condition="'%(SubmoduleConfiguration.Branch)' != '' AND
+                                       '%(SubmoduleConfiguration.SourceBuildAutoUpdate)' == 'true'" />
```

Comment:
Why not just change the condition to `'%(SubmoduleConfiguration.SourceBuildAutoUpdate)' != 'false'`?

Reply:
Depends on what we want to be truthy... I don't have a strong opinion here. As long as nobody tries to use `0`, `no`, or `null` or something, `!= 'false'` is fine and removes a few lines of code. I'll go ahead with changing it for now.